app/workers/alert_mailer_worker.rb

```diff
@@ -3,14 +3,14 @@
 class AlertMailerWorker
   include Sidekiq::Worker
 
-  def self.schedule_email(alert:, user:)
-    perform_async(alert.id, user.id)
+  def self.schedule_email(alert)
```

Comment:
For consistency in the way workers are set up, we can leave alert as a named parameter.